decidim-api/lib/decidim/api/test.rb

```diff
@@ -1,3 +1,3 @@
 # frozen_string_literal: true
 
-require "decidim/api/test/shared_examples/statistics_examples"
+Dir["#{__dir__}/test/**/*.rb"].each { |f| require f }
```

Comment:
I'd expect to have them explicit as in `decidim-core/lib/decidim/core/test.rb` - or viceversa, to do the same dynamic require there.

Reply:
I'm happy with either solution to be honest.

Reply:
The main idea is that i do not know how many examples i have. Also this is the way we load the shared examples for capybara

Reply:
Ok, we can leave it as is, no problem!